### PR TITLE
kubectl rollback: remove duplicated code for printing pod template

### PR DESCRIPTION
**Pull request by seans3**
* Removes duplicated pod template printing code
* Use `printTemplate` in history.go
* Removes dependencies on internal versions of core resources

Helps Address:
https://github.com/kubernetes/kubectl/issues/83

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/rollback.go [hunk 1]
```diff
@@ -36,12 +36,10 @@ import (
 	"k8s.io/apimachinery/pkg/util/strategicpatch"
 	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/client-go/kubernetes"
-	api "k8s.io/kubernetes/pkg/apis/core"
-	apiv1 "k8s.io/kubernetes/pkg/apis/core/v1"
 	kapps "k8s.io/kubernetes/pkg/kubectl/apps"
 	"k8s.io/kubernetes/pkg/kubectl/scheme"
 	sliceutil "k8s.io/kubernetes/pkg/kubectl/util/slice"
-	printersinternal "k8s.io/kubernetes/pkg/printers/internalversion"
+
```
Comment (apelisse) [inline pkg/kubectl/rollback.go]: drop empty line?

Comment (apelisse) [reply pkg/kubectl/rollback.go]: Or maybe not ...

File: pkg/kubectl/rollback.go [hunk 2]
```diff
@@ -247,15 +238,7 @@ func simpleDryRun(deployment *appsv1.Deployment, c kubernetes.Interface, toRevis
 	sliceutil.SortInts64(revisions)
 
 	template, _ := revisionToSpec[revisions[len(revisions)-2]]
-	buf := bytes.NewBuffer([]byte{})
-	buf.WriteString("\n")
```
Comment (apelisse) [inline pkg/kubectl/rollback.go]: This one has an extra `\n`. I don't think it matters ...

**Review by apelisse — COMMENTED** [review]
LGTM

**Review by soltysh — APPROVED** [review]
/lgtm

**Review by juanvallejo — APPROVED** [review]

**Pull request merged**